device_services/generic/scene_control.md

```diff
@@ -8,20 +8,23 @@ The service represents a device which can be used to control scenes, such as a r
 
 ## Interfaces
 
-| Type | Interface            | Value type | Description                                                            |
-|------|----------------------|------------|------------------------------------------------------------------------|
-| in   | cmd.scene.get_report | null       | Requests the currently active scene.                                   |
-| in   | cmd.scene.set        | string     | Sets the scene to the one provided in the value.                       |
-| out  | evt.scene.report     | string     | Event is generated whenever scene button is pressed on the controller. |
+| Type | Interface            | Value type | Description                                                                                                           |
+|------|----------------------|------------|-----------------------------------------------------------------------------------------------------------------------|
+| in   | cmd.scene.get_report | null       | Requests the currently active scene.                                                                                  |
+| in   | cmd.scene.set        | string     | Sets the scene to the one provided in the value.                                                                      |
+| out  | evt.scene.report     | string     | Event is generated whenever scene button is pressed on the controller.                                                |
+| out  | evt.lvl.report       | int        | Reports the level value. See the [`min_lvl`](#service-properties) and [`max_lvl`](#service-properties) for reference. |
 
 > Devices designed only as scene controllers may support only `evt.scene.report` interface and send report whenever a physical button is pressed.
 > Devices designed only as scene activators may support only `cmd.scene.set` interface and can be used to activate a scene by sending a command.
 
 ## Service properties
 
-| Name         | Type      | Example                                            | Description               |
-|--------------|-----------|----------------------------------------------------|---------------------------|
-| `sup_scenes` | str_array | `["1.key_pressed_1_time", "2.key_pressed_1_time"]` | List of supported scenes. |
+| Name         | Type      | Example                                             | Description                                     |
+|--------------|-----------|---------------------------------------|---------------------------------------------------------------|
+| `min_lvl`    | int       | `0`                                   | A minimum reportable level value for evt.lvl.report interface.|
+| `max_lvl`    | int       | `254`                                 | A maximum reportable level value for evt.lvl.report interface.|
+| `sup_scenes` | str_array | `["key_1_pressed", "key_2_pressed"]`  | List of supported scenes.                                     |
```

Comment:
Shouldn't we add transition time for future use? ZCL says about that parameter

Reply:
We can add it when necessary